setup.py

```diff
@@ -1,26 +1,24 @@
-#!/usr/bin/env python
-import os
-from distutils.core import setup
-
-setup(
-    name='osmo_jupyter',
-    version='0.0.1',
-    author='Osmo Systems',
-    author_email='dev@osmobot.com',
-    description='Tools for accessing and processing OsmoBot data in an experimental context',
-    url='https://www.github.com/osmosystems/python-jupyter-tools',
-    packages=['osmo_jupyter'],
-    install_requires=[
-        'pandas',
-        'numpy',
-        'scipy',
-        'pymysql',
-        'petl',
-        'intervaltree',
-        'requests',
-        'plotly==2',
-    ],
-    test_suite='pytest',
-    tests_require=['pytest'],

```

Comment:
changes in this file are mainly line endings (sorry! shouldn't happen again) and removing the `test_suite`, `tests_require`, and unused `os` import.

Reply:
👍